shared/teams/channel/header.tsx

```diff
@@ -62,6 +71,27 @@ const HeaderTitle = (props: HeaderTitleProps) => {
     </Kb.Box2>
   )
 
+  const onDeleteChannel = () => {
+    dispatch(nav.safeNavigateUpPayload())
+    dispatch(TeamsGen.createDeleteChannelConfirmed({conversationIDKey, teamID}))
+  }
+
+  const menuItems: Array<Kb.MenuItem> = [
+    {onClick: () => {}, title: 'Audience stats'},
+    // Not including settings here because there's already a settings tab below and plumbing the tab selection logic to here would be a real pain.
+    // It's included in the other place this menu appears.
```

Comment:
Isn't it just
```typescript
const onNavToSettings = () =>
    dispatch(
      nav.safeNavigateAppendPayload({
        path: [{props: {...props, selectedTab: 'settings' as const}, selected: 'teamChannel'}],
      })
    )
```

Reply:
That would put two of this page on top of each other, which would make the back button go back to the same page, which I don't like.